### ISSUE TITLE
schedule job N times 

### ISSUE THREAD

**Issue by heisen273** [dbader/schedule#142]
Have anyone achieved success in scheduling job only certain amount of times? 
I've seen example from docs, 'execute job only once', but  if there's any approach if I need to schedule job to run only, for example, 5 times?

**Comment by talaniz** [comment]
Wouldn't you be able to do this by implementing a counter? Using the example in the docs--

    import schedule
    import time

    def job():
        print("I'm working...")

    schedule.every(10).minutes.do(job)
    schedule.every().hour.do(job)
    schedule.every().day.at("10:30").do(job)
    schedule.every().monday.do(job)
    schedule.every().wednesday.at("13:15").do(job)

    job_counter = 0

    while job_counter >= 5:
        schedule.run_pending()
        time.sleep(1)

**Comment by heisen273** [comment]
@talaniz – first of all, thank you very much for your suggestion. I think I already tried this approach.

As per your code: in `while` i think you've meant `<=` instead of `>=`.
**however!** 
Have you even tried to running your code ? 

Please, see attached screenshot with code execution results.

![job_counter](https://user-images.githubusercontent.com/13763176/26860789-82dfc196-4b48-11e7-86d2-beb992d44754.png)

**Comment by heisen273** [comment]
**update**
Anyone who would like to achieve N-time job functionality, – please see #39 ; very efficient solution from @Aluriak

**Comment by guilhermecgs** [comment]
This would be a nice feature... Too bad it was not merged into master
